### ISSUE TITLE
Can we improve resource function code template

### ISSUE THREAD

**Issue by shafreenAnfar** [ballerina-platform/ballerina-lang#35490]
The following is what we generate now.

<img width="496" alt="Screen Shot 2022-03-17 at 10 27 06 AM" src="https://user-images.githubusercontent.com/6178058/158740635-c40e38c6-8877-47cf-ba87-c77442783b51.png">
But instead what if we do the below ?

<img width="517" alt="Screen Shot 2022-03-17 at 10 30 08 AM" src="https://user-images.githubusercontent.com/6178058/158740645-ee80f83a-4c24-489e-b9ff-83c0f78d1c5c.png">

**Comment by sameerajayasoma** [comment]
Should have `resource` instead of `path`?

**Comment by nadeeshaan** [comment]
> Should have `resource` instead of `path`?

Good point actually. The only concern I'm having with that approach is that we have the resource keyword in the signature already and whether the user finds it confusing in the beginning.

**Comment by shafreenAnfar** [comment]
> Good point actually. The only concern I'm having with that approach is that we have the resource keyword in the signature already and whether the user finds it confusing in the beginning.
Yeah, and also we don't have any concept of resource name or anything like that. All we capture is the URL to uniquely locate the resource.
